server/api/main.py

```diff
@@ -282,6 +275,20 @@ async def _verify_log_collection_started_on_startup(
     :param start_logs_limit: Semaphore which limits the number of concurrent log collection tasks
     """
     db_session = await fastapi.concurrency.run_in_threadpool(create_session)
+    log_collection_cycle_tracker = server.api.utils.time_window_tracker.TimeWindowTracker(
+        key=server.api.utils.time_window_tracker.TimeWindowTrackerKeys.log_collection,
+        max_window_size_seconds=min(
+            int(config.log_collector.api_downtime_grace_period),
+            int(config.runtime_resources_deletion_grace_period),
+        ),
```

Comment:
I think the comment should stay (it was in the `get_runs` call):
```
                # We take the minimum between the api_downtime_grace_period and the
                # runtime_resources_deletion_grace_period because we want to make sure that we don't miss any runs which
                # might have reached terminal state while the API was down, and their runtime resources are not deleted
```